bundles/org.openhab.binding.innogysmarthome/src/main/java/org/openhab/binding/innogysmarthome/internal/handler/InnogyBridgeHandler.java

```diff
@@ -110,7 +111,8 @@
     private @Nullable DeviceStructureManager deviceStructMan;
     private @Nullable String bridgeId;
     private @Nullable ScheduledFuture<?> reinitJob;
-    private @NonNullByDefault({}) InnogyBridgeConfiguration bridgeConfiguration;
+    private @NonNullByDefault({})
```

Comment:
Please revert this format change. Single annotations are on the same line.

Reply:
I changed it back.